### ISSUE TITLE
Can't authorize a device

### ISSUE THREAD

**Issue by ivanhjc** [getlantern/lantern#5756]
I've authorized 3 devices and couldn't authorized a fourth one. Is there a limitation as to the number of devices Lantern can authorize?

**Comment by mikedchavez1010** [comment]
Yes, at most 3 devices can be authorized. You can either buy another copy with a different email address or start lantern in its proxy server mode by appending --addr 0.0.0.0:8787 to lantern.exe on Windows or lantern on Linux.

On Windows, append the parameter above by modifying the shortcut link's properties (Shortcut -> Target). Then set the HTTP(S) proxy of your fourth device to the IP address of the PC running lantern with port 8787. On iOS, this can be set in the Settings -> WIFI option or by installing Wingy to manage the system proxies.

**Comment by mikedchavez1010** [comment]
Please refer to #5772

**Comment by denzel1894** [comment]
but how to authorize the other device, I can't authorize the device....
